### PR TITLE
add get All Shops in controller and service 

### PR DESCRIPTION
**Pull request by bavanya**

### CODE REVIEW

File: backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/service/VirtualQueueService.java [hunk 1]
```diff
@@ -40,4 +43,11 @@ public CreateShopResponse createShop(CreateShopRequest createShopRequest) {
 	public GetTokensResponse getTokens(UUID shopId) {
 		return new GetTokensResponse(shopId, virtualQueueRepository.getTokens(shopId));
 	}
+
+	public GetAllShopsResponse getAllShops() {
+		List<Shop> shopList = virtualQueueRepository.getAllShops();
+		GetAllShopsResponse getAllShopsResponse = new GetAllShopsResponse();
+		getAllShopsResponse.setShopList(shopList);
+		return getAllShopsResponse;
```
Comment (sdumka) [inline backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/service/VirtualQueueService.java:51]: These lines can be clubbed into one.

File: backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/proto/GetAllShopsResponse.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+import com.google.gpay.virtualqueue.backendservice.model.Shop;
+
+import lombok.Data;
+import lombok.Setter;
+
+@Data
+@Setter
+public class GetAllShopsResponse {
+    private List<Shop> shopList;
+}
```
Comment (sdumka) [inline backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/proto/GetAllShopsResponse.java:27]: Name it shops instead of shopList?

Comment (bavanya) [reply backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/proto/GetAllShopsResponse.java:27]: Done

File: backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/service/VirtualQueueService.java (hunk 1 above)
Comment (bavanya) [reply backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/service/VirtualQueueService.java]: Okay done

File: backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/service/VirtualQueueService.java [hunk 3]
```diff
@@ -40,4 +43,9 @@ public CreateShopResponse createShop(CreateShopRequest createShopRequest) {
 	public GetTokensResponse getTokens(UUID shopId) {
 		return new GetTokensResponse(shopId, virtualQueueRepository.getTokens(shopId));
 	}
+
+	public GetAllShopsResponse getAllShops() {
+		GetAllShopsResponse getAllShopsResponse = new GetAllShopsResponse();
+		return getAllShopsResponse.setShops(virtualQueueRepository.getAllShops());
```
Comment (sdumka) [inline backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/service/VirtualQueueService.java:49]: Club these 2 as well. 
return  new GetAllShopsResponse().setShops(virtualQueueRepository.getAllShops());
Similarly in the other PR.

Comment (bavanya) [reply backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/service/VirtualQueueService.java]: Ok Done

**Pull request merged**